src/ConstrainedAstrometryModel.cc

```diff
@@ -232,5 +219,14 @@ std::shared_ptr<TanSipPix2RaDec> ConstrainedAstrometryModel::produceSipWcs(CcdIm
     Point tangentPoint(proj->getTangentPoint());
     return std::make_shared<TanSipPix2RaDec>(cdStuff, tangentPoint, &sip);
 }
+
+AstrometryMapping *ConstrainedAstrometryModel::findMapping(CcdImage const &ccdImage) const {
+    auto i = _mappings.find(ccdImage.getHashKey());
+    if (i == _mappings.end())
```

Comment:
`i->first`?

Reply:
I'm confused by this comment. What are you suggesting be changed to `i->first`? This is the standard way to check that a thing isn't in a `std::map`.